ui/js/animations.js

```diff
@@ -27,6 +27,9 @@ var Animations = (function() {
   var initialized = false;
   var wiperSpeed;
 
+  // Redraw after every 5 frames.
+  var frameSkipRate = 5;
```

Comment:
Ashley tested this and pointed out that the frame skipping let to a bit of jumpiness with things like the wipers.  She said 3 and 1 values worked better for her.  @dbanda can you consider changing this to 3?